source/octf/interface/TraceConverter.cpp

```diff
@@ -126,11 +128,17 @@ std::shared_ptr<const google::protobuf::Message> TraceConverter::convertTrace(
 
         auto protoFsMeta = m_evFsMeta->mutable_filesystemmeta();
         protoFsMeta->set_refsid(event->ref_sid);
-        protoFsMeta->set_fileid(event->file_id);
         protoFsMeta->set_fileoffset(event->file_offset);
         protoFsMeta->set_filesize(event->file_size);
         protoFsMeta->set_partitionid(event->partition_id);
 
+
+        nodeId = protoFsMeta->mutable_fileid();
+        nodeId->set_fileid(event->file_id.file_id);
+        timestamp = nodeId->mutable_creationdate();
+        timestamp->set_sec(event->file_id.ctime.tv_sec);
```

Comment:
Does types fit each other?

Reply:
timespec64 has 
long tv_nsec;
time64_t	tv_sec;
where
typedef __s64 time64_t;
So 2x u64 should be fine?